docs/logging.md

```diff
@@ -0,0 +1,134 @@
+# Logging
+
+Thanos supports request logging via flags.
+
+Components are configured using `--request.logging-config-file` to reference to the configuration file or `--request.logging-config` as YAML directly.
+
+## Configuration
+
+Configuration can be supplied for either gRPC, HTTP or both. Options can be supplied globally, which is applied to both gRPC and HTTP, or individually to either or both protocols.
```

Comment:
```suggestion
Configuration can be supplied for either gRPC, HTTP or both. Options can be supplied globally, which is applied to both gRPC and HTTP, or individually to either of both protocols.
```

Reply:
I think my wording is correct here actually but I see it can be confusing. Does this work for you:

```
Configuration can be supplied globally which applies to both `grpc` and `http`. Alternatively both `http` and/or `grpc` can be configured independently.
```

Reply:
Sounds good!